src/store.js

```diff
@@ -0,0 +1,21 @@
+import { createStore } from 'redux'
+// import applyMiddleware from "redux/src/applyMiddleware";
+
+import reducer from './reducers'
+//
+// const logger = (store)=>(next)=>(action)=>{
+//     console.log(action);
+//     next(action)
+// }
+// // const middleware = applyMiddleware(logger);
+
+ // const store =  createStore(reducer,1,middleware)
+const store =  createStore(reducer)
+
+
+store.subscribe(()=>{
```

Comment:
`const store = createStore(reducers /*Original state*/
, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())`

instead of this line will link to [Redux dev-tools](https://github.com/zalmoxisus/redux-devtools-extension), An extension for chrome and firefox, that can show actions, state before and after, with the ability of going back or running actions directly from console for testing.